sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -550,6 +550,8 @@ object FunctionRegistry {
     expression[BitAndAgg]("bit_and"),
     expression[BitOrAgg]("bit_or"),
     expression[BitXorAgg]("bit_xor"),
+    expression[BitwiseGet]("bit_get"),
```

Comment:
All the other databases seemed to have the `getbit` only. Perhaps it's better to not pollute the function list with aliases.

Reply:
`bit_get` conforms to the style of spark before, and `getbit` is a commonly used name for other databases.

Reply:
I assume the point is to add functions that other DBs have - this seems pretty niche. If so, it needs to be `getbit` no? how commonly used is this?

Reply:
I think using `getbit` can increase the compatibility of spark with other mainstream databases. Sometimes, people want migrate from other database to Spark+Hive.